app/reducers/computed/formatResult.js

```diff
@@ -1,9 +1,9 @@
 import bytes32ToNum from './bytes32ToNum'
 
 const formatResult = (result, blockTimestamp) => {
-  const unixTime = new Number(blockTimestamp) * 1000
 
-  return unixTime > 0 ? new String(bytes32ToNum(result)) : '--'
+  const unixTime = new Number(blockTimestamp) * 1000
+  return unixTime > 0 ? new String(bytes32ToNum(result) / 10 ** 18) : '--'
```

Comment:
consider moving the `--` formatting to a view (component level) .. state should generally be data only and not conflated with the way things are rendered. For example, this would be undefined or null in state, but the view would have some condition, if undefined, render `"--"` on the UI.